src/ManagementRoomOutput.ts

```diff
@@ -94,6 +95,10 @@ export default class ManagementRoomOutput {
      * @param isRecursive Whether logMessage is being called from logMessage.
      */
     public async logMessage(level: LogLevel, module: string, message: string | any, additionalRoomIds: string[] | string | null = null, isRecursive = false): Promise<any> {
+        if (level === LogLevel.ERROR) {
+            // Propagate any error to Sentry, for sorting and monitoring.
+            Sentry.captureMessage(`${module}: ${message}`, 'error');
```

Comment:
![image](https://user-images.githubusercontent.com/50846879/199728820-af835363-a77c-4962-b6f9-e41f731f71ca.png)

Another problem here is that if you do create an error, we can't give it to `logMessage` so we don't get any stack trace. I also don't understand how the breadcrumbs is being generated around this error, do you know how it is getting that? Maybe there is some magic already in matrix-bot-sdk?

Reply:
> Another problem here is that if you do create an error, we can't give it to logMessage so we don't get any stack trace. 

We could introduce `logError` specifically to log:

- an `Error`;
- any human-readable context

What do you think?

>  I also don't understand how the breadcrumbs is being generated around this error, do you know how it is getting that? Maybe there is some magic already in matrix-bot-sdk?

I believe that Sentry is doing magic to intercept `console`, http requests and other events at Node-level.

Reply:
That would work, though it will need reworking with afterwards https://github.com/matrix-org/mjolnir/pull/398#discussion_r1015280542

Reply:
I've added a candidate `logError` in the latest version.